Sample Applications/WPFGallery/Views/DashboardPage.xaml

```diff
@@ -16,8 +16,8 @@
       </Border.Background>
     </Border>
         <StackPanel Grid.Row="0" VerticalAlignment="Center">
-            <TextBlock Foreground="White" Text="WPF Win11 Theme Gallery" FontWeight="Bold" FontSize="28" Margin="30 0 0 0" Padding="30 0 0 5"/>
-            <TextBlock Foreground="White" FontSize="16" Margin="30 0 0 0" Padding="30 0 0 0" FontWeight="Normal">Application to showcase Win11 Theme in WPF</TextBlock>
+            <TextBlock Foreground="White" Text="WPF Fluent Theme Gallery" FontWeight="Bold" FontSize="28" Margin="30 0 0 0" Padding="30 0 0 5"/>
```

Comment:
Making sure, if we want this to be WPF Fluent Theme Gallery or WPF Gallery Application.

Reply:
Made the changes